### ISSUE TITLE
Incorrect object reference in Gantt Chart

### ISSUE THREAD

**Issue by cilerler** [plantuml/plantuml#1809]
I've encountered an issue with the Gantt chart where it treats `T` and `Z` as the same object, despite being defined separately using the `as` keyword. Below is the snippet demonstrating the problem:

```plantuml
@startgantt
[Prototype design] as [D] requires 15 days
[Test prototype] as [T] requires 10 days
[T] starts at [D]'s end
[Test prototype] as [Z] requires 3 days
@endgantt
```

## Expected behavior:
T and Z should be recognized as distinct objects.

## Actual behavior
T and Z are treated as the same object.

**Comment by The-Lum** [comment]
_[Just for the record]_
See similar report/request here:
- https://forum.plantuml.net/12176/gantt-aliases-differently-other-diagrams-possible-same-label

Regards,
Th.

**Comment by cilerler** [comment]
@The-Lum, you're really good at finding these 👏🏻😎. I missed that one, thanks for pointing it out.

**Comment by The-Lum** [comment]
Hi @cilerler,

FYI that is now implemented on the [last release version V1.2024.6](https://github.com/plantuml/plantuml/releases/tag/v1.2024.6).

Thanks to @arnaudroques,
Regards,
Th.

**Comment by cilerler** [comment]
Thanks @The-Lum for notifying, and @arnaudroques making it happen. 🤗 
I confirm that it is working.
